xapian-core/cluster/euclidian_sim.cc

```diff
@@ -77,29 +78,40 @@ EuclidianDistance::similarity(TermListGroup tlg,
 		wt = tf*idf;
 		docvec2.push_back(make_pair(*titer,wt));
 	}
-	
+
 	vector<pair<string, double> >::iterator ta;
 	vector<pair<string, double> >::iterator tb;
 	double result = 0;
-	
-	// Finding euclidian distance between the above document vectors	
+
+	// Finding euclidian distance between the above document vectors
 	for(ta = docvec1.begin(); ta != docvec1.end(); ta++) {
 		bool found = false;
 		double sum = 0;
 		for(tb = docvec2.begin(); tb != docvec2.end(); tb++) {
 			if(tb->first == ta->first) {
-				found=true;
+				found = true;
 				break;
 			}
 		}
-		
-		if(found) {
```

Comment:
We put spaces after `if` (also `for`, above, and similar), before the brackets. So:

```c++
if (found) {
```